packages/scripts/package.json

```diff
@@ -73,6 +73,7 @@
 		"npm-package-json-lint": "^6.4.0",
 		"npm-packlist": "^3.0.0",
 		"postcss": "^8.4.5",
+		"postcss-import": "16.1.0",
```

Comment:
```suggestion
		"postcss-import": "^16.1.0",
```

Reply:
Thank you for the review. I'll make the suggested adjustements in the next few days.